src/nixos/machines/lengo/config/disks.nix

```diff
@@ -16,50 +16,58 @@ let
 	inherit (lib) mkMerge;
 
 	diskoDevice = "/dev/disk/by-id/nvme-WD_PC_SN740_SDDPMQD-512G-1101_2335R1406872";
+	keyDevice = "/dev/disk/by-id/mmc-NCard_0x23904944";
 	swapSize = "60G";
+	impermanentSize = "5G"; # Size of the impermanent filesystem during impermanence
 in mkMerge [
 	{
-		age.secrets.lengo-disks-password.file = ../secrets/lengo-disks-password.age; # Supply password for disk encryption
+		age.secrets.lengo-disks-password.file = "${self.outPath}src/nixos/machines/lengo/secrets/lengo-disks-password.age"; # Supply password for disk encryption
```

Comment:
Pending to replace all `../path/somewhere` with `${self.outPath}/path/somewhere` to replace relative paths for absolute paths.